src/RealtimeEffectPanel.cpp

```diff
@@ -293,10 +294,63 @@ namespace
          event.SetSourceIndex(mSourceIndex);
          event.SetTargetIndex(mTargetIndex);
          event.SetEventObject(this);
-         wxPostEvent(target, event);
+         target->GetEventHandler()->ProcessEvent(event);
```

Comment:
Perhaps then the name of this function should change to `ProcessDragEvent` because it no longer delays the dispatch.